keyvi/src/cpp/dictionary/fsa/automata.h

```diff
@@ -40,6 +40,7 @@
 #include "dictionary/util/endian.h"
 #include "dictionary/fsa/internal/intrinsics.h"
 #include "dictionary/dictionary_merger_fwd.h"
+#include "dictionary/util/keyvi_file.h"
```

Comment:
I think dictionary::fsa is the right place. Can we name it automata_file, I think that fits better.